README.md

```diff
@@ -62,7 +62,8 @@ Releases of twilio-video.js are hosted on a CDN, and you can include these
 directly in your web app using a &lt;script&gt; tag.
 
 ```html
-<script src="//media.twiliocdn.com/sdk/js/video/latest/twilio-video.min.js"></script>
+<script src="//media.twiliocdn.com/sdk/js/video/releases/2.10.0/twilio-video.min.js"></script>
```

Comment:
```suggestion
<script src="//media.twiliocdn.com/sdk/js/video/latest/twilio-video.min.js"></script>
```

Reply:
@manjeshbhargav - we only pin docs, not the sdk, so we cannot use `latest` here